### PR TITLE
nano: Add a plus variant with more features

### PR DESCRIPTION
**Pull request by hnyman**
Nano is by default built as "tiny" with most features disabled.

Add a new nano-plus variant that enables selected additional features in the build config: 
multiple files (multibuffer), justify, nanorc, help. 
Also some key bindings get enabled as the "tiny" configure option is removed.

Submitted as PR for CI testing...

Maintainer: me 
Run tested: ipq806x/R7800

### CODE REVIEW

File: utils/nano/Makefile [hunk 1]
```diff
@@ -22,18 +22,31 @@ PKG_CPE_ID:=cpe:/a:gnu:nano
 
 PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
```
Comment (neheb) [inline utils/nano/Makefile:25]: wasn't there an upstream commit that already does this?

Comment (hnyman) [reply utils/nano/Makefile:25]: you are apparently right. Since 2019 there is a default line for that purpose. 
I will modify and test.

Comment (alexeys85) [reply utils/nano/Makefile:25]: Is it possible to include unicode support in that plus variant?

**Comment by alexeys85** [comment]
Is it possible to include unicode support in that plus variant?

**Comment by hnyman** [comment]
It might be, but that would then increase the installed size quite a lot, I guess. (including the possible add-on packages)

**Comment by hnyman** [comment]
I tested, and Unicode/UTF-8 support only adds 2 kB to the plus variant, as we already provide libncursesw and libncurses from the same file.

I added it to the plus variant.

**Pull request merged**